### ISSUE TITLE
Add `qs2` fomat to `save_object`

### ISSUE THREAD

**Issue by stemangiola** [stan-dev/cmdstanr#1040]
`qs2` is a very fast format for saving and reading data. For buge models, `fit$save_object()` is slow in `rds` format.

It would be great to ass the `qs2` (or `qs`) option to  `save_object`.

Thanks!

**Comment by jgabry** [comment]
Yeah, it's definitely slow for big models. We do have a section in the vignette (contributed by @wlandau) showing how to do it with `qs::qsave()`, but there's not a built-in option at the moment: 

https://mc-stan.org/cmdstanr/articles/cmdstanr.html#saving-fitted-model-objects

**Comment by jgabry** [comment]
For now https://github.com/stan-dev/cmdstanr/pull/1042 will point to this section of the vignette from the documentation of `save_object`.
